helpers/database.go

```diff
@@ -4,28 +4,36 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"io/ioutil"
+	"log"
+
 	_ "github.com/jackc/pgx/v4"
 	_ "github.com/jackc/pgx/v4/stdlib"
-	"log"
 )
 
-const TestDataPrefix = "perf-%"
-
-
-func OpenDbConnections(ccdbConnection string, uaadbConnection string)(ccdb, uaadb *sql.DB, ctx context.Context){
-
-	ccdb, err := sql.Open("pgx", ccdbConnection)
+func OpenDbConnections(testConfig Config) (ccdb, uaadb *sql.DB, ctx context.Context) {
+	ccdb, err := sql.Open("pgx", testConfig.CcdbConnection)
 	checkError(err)
 
-	uaadb, err = sql.Open("pgx", uaadbConnection)
-	checkError(err)
+	if testConfig.UaadbConnection != "" {
+		uaadb, err = sql.Open("pgx", testConfig.UaadbConnection)
+		checkError(err)
+	}
 
 	ctx = context.Background()
 
 	return
 }
 
-func CleanupTestData(ccdb, uaadb *sql.DB, ctx context.Context) {
+func ImportStoredProcedures(ccdb *sql.DB, ctx context.Context) {
+	content, err := ioutil.ReadFile("../scripts/pgsql_functions.sql")
+	if err != nil {
+		log.Fatal(err)
+	}
+	ExecuteStatement(ccdb, ctx, string(content))
```

Comment:
With this command, it means that we don't need to run all CREATE FUNCTION in Query console at first, then use the created functions in postgres?

Reply:
Yes, the content of the SQL-file will be imported automatically. This only takes a few milliseconds and with "CREATE OR REPLACE FUNCTION" it's even possible to change the function implementation.